x-pack/plugins/security_solution/scripts/endpoint/resolver_generator_script.ts

```diff
@@ -165,6 +165,12 @@ async function main() {
       type: 'boolean',
       default: false,
     },
+    logsEndpoint: {
```

Comment:
curious: why the option? why not just always include action/responses?

Reply:
So that I add fake data to the indices only with a flag and not when adding to fleet indices. This way I can test the API logic that handles the case when the new indices do not exist (older endpoint or package) but fleet indices do.

Reply:
ahh. Ok. so maybe clarify the docs (describe below) in its usage. mention that by default, no endpoint log action are added, but wiht this option, those will be added. maybe mention the version when we started to support it (7.16?)

Reply:
done 881900bbc5ab21cc17353341cda34281fbb9f346